tests/test_vfs.py

```diff
@@ -146,4 +147,26 @@ def test_path(self):
         self.assertEqual(
             self.c.path,
             '/vsitar/{path}/testing/coutwildrnp.shp'.format(
-                path=self.path))
+                path=self.path))    
+
+        
+def test_open_http():
```

Comment:
@youngpm Could you flag this test with @pytest.mark.network so we can skip it with $ pytest -m "not network" to make offline development easier? Same with the test below.